app/person.rb

```diff
@@ -1,9 +1,16 @@
 # Represents a student, inheriting from the Person class.
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Subclasses must implement the correct_name method.'
+  end
+end
+
+class Person < Nameable
   attr_accessor :name, :age
```

Comment:
![image](https://github.com/martinkarugaba/ruby-oop-school-library/assets/93039359/8ee0e273-47bc-4a1f-8f21-db40651d440d)

- [ ] It would be great if you fix the unexpected behavior of your app which you can see in the image above that it is giving me **"Unkown"** all the time 😨 Kindly fix it to make your app work as expected 👍